### PR TITLE
Steve/postgres database

### PR DESCRIPTION
**Pull request by MHDGhanemBalhawan**
Changing the database from MongoDB to Postgres and adding the script file for creating the database.

### CODE REVIEW

File: backend/route/router.js [hunk 1]
```diff
@@ -1,19 +1,82 @@
...
+    } else {
+      res.status(200).send(result.rows);
+    }
+  });
 });
 
-router.get("/events/:id", (req, res) => {
+router.get("/:id", (req, res) => {
+  // const event = _.find(events, event => event.id === req.params.id);
+  // if (event) {
```
Comment (dahfool) [inline backend/route/router.js]: @MHDGhanemBalhawan Please remove, code you commented out.

File: backend/data/events.model.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const mongoose = require("mongoose");
+
+const UserSchema = mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
```
Comment (dahfool) [inline backend/data/events.model.js]: @MHDGhanemBalhawan Are you using mongoose with postgress

File: backend/route/router.js [hunk 3]
```diff
@@ -1,19 +1,82 @@
...
+  //   res.json(event);
+  // } else {
+  //   res.send(`Event with id ${req.params.id} not found.`);
+  // }
   const id = req.params.id;
```
Comment (dahfool) [inline backend/route/router.js:38]: @MHDGhanemBalhawan Please remove, code you commented out

File: backend/data/events.model.js (hunk 2 above)
Comment (dahfool) [inline backend/data/events.model.js]: @MHDGhanemBalhawan We using mongoose with postgress ?

**Comment by dahfool** [comment]
@MHDGhanemBalhawan mongod.lock file need be deleted.

File: backend/route/router.js [hunk 4]
```diff
@@ -1,19 +1,82 @@
...
-const events = require("../events.json");
 
-router.get("/home", (req, res) => {
-  res.send("it is working");
+const { Pool } = require("pg");
+const events = require("../data/events");
+const bodyParser = require("body-parser");
+
+// parse the post request as json
+var json_body_parser = bodyParser.json();
```
Comment (dahfool) [inline backend/route/router.js:8]: es6, we should use either const or let, not var

File: backend/data/events.model.js (hunk 2 above)
Comment (alismith-dev) [reply backend/data/events.model.js]: It looks like we don't have a dependency on mongoose, so I think this file can be removed.

File: backend/route/router.js [hunk 5]
```diff
@@ -1,19 +1,82 @@
...
+  // } else {
+  //   res.send(`Event with id ${req.params.id} not found.`);
+  // }
   const id = req.params.id;
   let result = events.filter(item => item.id === id);
   res.json(result);
 });
 
+router.post("/", (req, response) => {
+  // console.log(req.body);
```
Comment (alismith-dev) [inline backend/route/router.js]: Similar to the comment above, can you remove this unused `console.log`

File: backend/server.js [hunk 6]
```diff
@@ -2,6 +2,21 @@ const express = require("express");
 const app = express();
 const router = require("./route/router");
 
+const events = require("./data/events.json");
+const lodash = require("lodash");
```
Comment (alismith-dev) [inline backend/server.js]: I don't see where these dependencies are used? If they are not needed can you remove them?

**Review by 40thieves — CHANGES_REQUESTED** [review]
A few things to clean up, but it looks like the right direction

**Review by 40thieves — APPROVED** [review]
LGTM! :shipit:

Nice work :)

**Review by anton-kozyr — APPROVED** [review]

**Review by anton-kozyr — APPROVED** [review]

**Pull request merged**